Makefile

```
@@ -279,6 +279,13 @@ python/format:
 	black $(PYTHON_FILES)
 	@ echo "[$@]: Successfully formatted Python files!"
 
+# Run pytests, typically for unit tests.
+pytest/%: | guard/program/pytest
+pytest/%:
+	@ echo "[$@] Starting Python tests"
+	pytest $*
```

To generalize further, so users can modify how pytest runs, it would be good for this to take `PYTEST_ARGS`"
```suggestion
	pytest $* $(PYTEST_ARGS)
```

---

ok